src/vellum/workflows/runner/runner.py

```diff
@@ -1016,6 +1019,33 @@ def _run_cancel_thread(self, kill_switch: ThreadingEvent) -> None:
                 )
                 return
 
+    def _run_timeout_thread(self, kill_switch: ThreadingEvent) -> None:
+        if not self._timeout:
+            return
+
+        if kill_switch.wait(timeout=self._timeout):
+            return
+
+        parent_context = WorkflowParentContext(
+            span_id=self._initial_state.meta.span_id,
+            workflow_definition=self.workflow.__class__,
+            parent=self._execution_context.parent_context,
+        )
+
+        self._emit_node_cancellation_events(
+            error_message=f"Workflow execution exceeded timeout of {self._timeout} seconds",
+            parent_context=parent_context,
```

Comment:
remove this kwarg actually. Should be the same for all usages of `_emit_node_cancellation_events`